compiler/rustc_typeck/src/check/method/probe.rs

```diff
@@ -1463,6 +1463,7 @@ impl<'a, 'tcx> ProbeContext<'a, 'tcx> {
                 TraitCandidate(trait_ref) => {
                     if let Some(method_name) = self.method_name {
                         // Some trait methods are excluded for arrays before 2021.
+                        // (`array.into_iter()` wants a slice iterator for compatability.)
```

Comment:
Typo: 'compatability' should be 'compatibility'

Reply:
Whoops, thanks!